### PR TITLE
Logic in service

### PR DESCRIPTION
**Pull request by Abigail-cloud**
Created a new folder called service and split the three services into different files.

### CODE REVIEW

File: userView/models/User.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  location: {
+    type: String,
+    trim: true,
+    maxlength: 20,
+    default: 'my city',
+  },
+})
+
+UserSchema.pre('save', async function () {
+  // console.log(this.modifiedPaths())
```
Comment (forestbaba) [inline userView/models/User.js]: Remove dead codes

File: userView/middleware/error-handler.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const { StatusCodes } = require ('http-status-codes')
+
+const errorHandlerMiddleware = (err, req, res, next) => {
+  const defaultError = {
+    statusCode: err.statusCode || StatusCodes.INTERNAL_SERVER_ERROR,
+    msg: err.message || 'Something went wrong, try again later',
+  }
+  if (err.name === 'ValidationError') {
+    defaultError.statusCode = StatusCodes.BAD_REQUEST
+    // defaultError.msg = err.message
```
Comment (forestbaba) [inline userView/middleware/error-handler.js]: Remove dead codes

File: userView/models/User.js (hunk 1 above)
Comment (Abigail-cloud) [reply userView/models/User.js]: I've removed the dead codes...

File: userView/middleware/error-handler.js (hunk 2 above)
Comment (Abigail-cloud) [reply userView/middleware/error-handler.js]: I've removed dead codes...

**Review by forestbaba — APPROVED** [review]
LGTM

**Pull request merged**